src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -1741,6 +1741,7 @@ public void BuildInDesignTimeMode ()
 			var proj = new XamarinAndroidApplicationProject () {
 				IsRelease = true,
 			};
+			proj.SetProperty ("_AndroidUseManagedDesignTimeResourceGenerator", "False");
```

Comment:
Ditto: if this is a DesignTime build, why disable the managed parser?

Should the DesignTimeBuild-related tests instead be run *twice*, with each parser?